acquire/acquire.py

```diff
@@ -248,6 +248,42 @@ def _run(cls, target, collector):
         pass
 
 
+@register_module("--sys")
+@local_module
+class Sys(Module):
+    DESC = "Sysfs files (live systems only)"
+    EXEC_ORDER = ExecutionOrder.BOTTOM
+
+    @classmethod
+    def _run(cls, target: Target, collector: Collector):
+        spec = [("dir", "/sys")]
```

Comment:
```suggestion
        if sys.platform == "win32":
            log.error("/sys collection can only run Unix-like systems! Skipping...")
            return

        spec = [("dir", "/sys")]
```
Maybe add something like this? Also to the proc one. Or maybe make it more specific rather than checking for win32, whatever you think is best :)

Reply:
Good one! Made the check somewhat more specific. I'd like to acquire `/proc` and `/sys` from WSL child targets in the future. Though, that should be a separate PR / ticket